hudi-cli/src/main/java/org/apache/hudi/cli/commands/HoodieLogFileCommand.java

```diff
@@ -104,7 +104,7 @@ public String showLogFileCommits(
           try {
             instantTime = n.getLogBlockHeader().get(HeaderMetadataType.INSTANT_TIME);
             if (instantTime == null) {
-              throw new Exception("Invalid instant time " + instantTime);
```

Comment:
just leave it as instantTime ? (also this change is different from what the PR says so lets not have it in this PR)

Reply:
@n3nash thanks for your comment, IMO, it's a minor change (which has no function impact) may be no need separate PR. 

For the minor change: The `instantTime` variable is known to be null.
```
"Invalid instant time " + null
```

Reply:
Right, I agree it's a minor change and doesn't need a new PR. But I don't think we really need this change

Reply:
@n3nash follow you, reverted.